src/pages/Dashboard/ParentComponent.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import ChildComponent from "./ChildComponent";
+
+const ParentComponent = () => {
```

Comment:
Please rename it to well define which should be helpful to understand the objective of this code or component.

Reply:
done